steps/feature-list-for-release-notes/0.0.1/step.yml

```diff
@@ -0,0 +1,44 @@
+title: Feature list for release notes
+summary: |
+  Creates a list of features gathered from git commits using given pattern.
+description: |
+  Feature descriptions are derived from commit messages. Using given pattern, one can create a list of the featurees based on previous commits. Also a alternative use would be to create links to appropriate features in the backlog system.
+website: https://github.com/maqoo/bitrise-step-feature-list-for-release-notes
```

Comment:
This URL: `https://github.com/maqoo/bitrise-step-feature-list-for-release-notes` does not exist, please update, since this URL will present by the workflow editor